ZeldaFullEditor/Gui/DungeonMain.cs

```diff
@@ -3031,54 +3108,54 @@ protected override CreateParams CreateParams
             get
             {
                 const int WS_EX_COMPOSITED = 0x02000000;
-                var cp = base.CreateParams;
-                cp.ExStyle |= WS_EX_COMPOSITED;
-                return cp;
+                var createParams = base.CreateParams;
+                createParams.ExStyle |= WS_EX_COMPOSITED;
+                return createParams;
             }
         }
 
         private void PrintRoomObjectsToolStripMenuItem_Click(object sender, EventArgs e)
         {
-            foreach (Room_Object o in this.activeScene.room.tilesObjects)
+            foreach (Room_Object roomObject in this.activeScene.room.tilesObjects)
             {
-                if (o.options == ObjectOption.Door)
+                if (roomObject.options == ObjectOption.Door)
                 {
-                    if (o is object_door d)
+                    if (roomObject is object_door doorObject)
                     {
-                        Console.WriteLine("n:" + d.name + ", door dir:" + d.door_dir + ", door pos:" + d.door_pos + ", door type:" + d.door_type);
+                        Console.WriteLine($"n:{doorObject.name}door dir:{doorObject.door_dir}, door pos:{doorObject.door_pos}, door type:{doorObject.door_type}");
                     }
                 }
                 else
                 {
-                    Console.WriteLine("n:" + o.name + ", w:" + o.width + ", h:" + o.height + ", L:" + o.Layer);
+                    Console.WriteLine($"n:{roomObject.name}, w:{roomObject.width}, h:{roomObject.height}, L:{roomObject.Layer}");
                 }
             }
         }
 
-        // TODO DISGUSTING, these "Contains" should instead become properties of the object itself
-        private void RemoveMasksObjectsToolStripMenuItem_Click(object sender, EventArgs e)
+		// TODO KAN REFACTOR DISGUSTING, these "Contains" should instead become properties of the object itself.
```

Comment:
nit: unclear what this is talking about and a little hyperbolic

Reply:
changed